[2022-05-23 20:16:40,464] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-09T00:00:00+00:00 [queued]>
[2022-05-23 20:16:40,530] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-09T00:00:00+00:00 [queued]>
[2022-05-23 20:16:40,531] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:16:40,531] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-05-23 20:16:40,531] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:16:40,613] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): second_task> on 2022-05-09 00:00:00+00:00
[2022-05-23 20:16:40,640] {standard_task_runner.py:52} INFO - Started process 3362 to run task
[2022-05-23 20:16:40,650] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-09T00:00:00+00:00', '--job-id', '856', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp0tyrvdgo', '--error-file', '/tmp/tmpnv9jp9jk']
[2022-05-23 20:16:40,651] {standard_task_runner.py:80} INFO - Job 856: Subtask second_task
[2022-05-23 20:16:40,843] {task_command.py:369} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-09T00:00:00+00:00 [running]> on host 061a475e6c14
[2022-05-23 20:16:41,137] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=gerson
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-09T00:00:00+00:00
[2022-05-23 20:16:41,138] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-23 20:16:41,139] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-05-23 20:16:41,158] {subprocess.py:85} INFO - Output:
[2022-05-23 20:16:41,160] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-05-23 20:16:41,161] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-23 20:16:41,295] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220509T000000, start_date=20220523T201640, end_date=20220523T201641
[2022-05-23 20:16:41,337] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-23 20:16:41,423] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
